### DISCUSSION TITLE
Does ungoogled-chromium slow down m1 mac?

### DISCUSSION THREAD

**Discussion by j0k4rX** [Eloston/ungoogled-chromium#1908] [Q&A] [answered]
Hi,
I recently switched to apple silicon m1 mac & a bit confused which browser to get.
so, my question is "does ungoogled-chromium have anything like: google-chrome (keystone) or whatever that'd slow down/brick my lap?"
Regards.

**Comment by networkException** [comment, marked as answer]
ungoogled-chromium does not come with any update detection on its own, the application will only use resources when running

**Reply by j0k4rX** [reply]
> ungoogled-chromium does not come with any update detection on its own, the application will only use resources when running

How is it compared to Brave browser?
Does it take less resources & is less power hungry than brave?

**Reply by networkException** [reply]
I assume they will be similar but you'd have to measure to find out for certain
